### ISSUE TITLE
install on latest scala & sbt - is it possible?

### ISSUE THREAD

**Issue by Pitometsu** [ensime/scala-debugger#367]
```
[warn] 	Note: Some unresolved dependencies have extra attributes.  Check that these dependencies exist with the requested attributes.
[warn] 		org.scala-debugger:sbt-scala-debugger:1.1.0-M3 (scalaVersion=2.12, sbtVersion=1.0)
```

**Comment by chipsenkbeil** [comment]
@Pitometsu, I haven't published the plugin for sbt 1.x. The plugin is currently available for 0.13.x for Scala 2.10/2.11/2.12.

**Comment by Pitometsu** [comment]
Thank you, will waiting on version for 1.x.
